README.md

```diff
@@ -1,58 +1,127 @@
 
-If you'd like to get help with the mod, check out the latest developments, or be notified when there's a new release,
-the Discord community might be for you! You can join the official server for my mods by clicking
-[here](https://jellysquid.me/discord).
+We have an [official Discord community](https://jellysquid.me/discord) for all of our projects. By joining, you can:
+- Get installation help and technical support with all of our mods 
+- Be notified of the latest developments as they happen
+- Get involved and collaborate with the rest of our team
+- ... and just hang out with the rest of our community.
 
 ---
 
-### Building from source
+### Building from sources
+
+Support is not provided for setting up build environments or compiling the mod. We ask that
+users who are looking to get their hands dirty with the code have a basic understanding of compiling Java/Gradle
+projects. The basic overview is provided here for those familiar.
 
-If you're hacking on the code or would like to compile a custom build of Phosphor from the latest sources, you'll want
-to start here.
+#### Requirements
 
-#### Prerequisites
+- JRE 8 or newer (for running Gradle)
+- JDK 8 (optional)
+  - If you neither have JDK 8 available on your shell's path or installed through a supported package manager (such as
+[SDKMAN](https://sdkman.io)), Gradle will automatically download a suitable toolchain from the [AdoptOpenJDK project](https://adoptopenjdk.net/)
+and use it to compile the project. For more information on what package managers are supported and how you can
+customize this behavior on a system-wide level, please see [Gradle's Toolchain user guide](https://docs.gradle.org/current/userguide/toolchains.html).
+- Gradle 6.7 or newer (optional)
+  - The [Gradle wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:using_wrapper) is provided in
+    this repository can be used instead of installing a suitable version of Gradle yourself. However, if you are building
+    many projects, you may prefer to install it yourself through a suitable package manager as to save disk space and to
+    avoid many different Gradle daemons sitting around in memory.
 
-You will need to install JDK 8 in order to build Phosphor. You can either install this through a package manager such as
-[Chocolatey](https://chocolatey.org/) on Windows or [SDKMAN!](https://sdkman.io/) on other platforms. If you'd prefer to
-not use a package manager, you can always grab the installers or packages directly from
-[AdoptOpenJDK](https://adoptopenjdk.net/).
+#### Building with Gradle
 
-On Windows, the Oracle JDK/JRE builds should be avoided where possible due to their poor quality. Always prefer using
-the open-source builds from AdoptOpenJDK when possible.
+Phosphor ses a typical Gradle project structure and can be built by simply running the default `build` task. After Gradle
```

Comment:
typo on sees

Reply:
fixed (it is actually supposed to say 'uses')